.env.development

```diff
@@ -1 +1,2 @@
+REACT_QUAY_APP_API_URL=http://localhost:8080
 REACT_APP_QUAY_DOMAIN=localhost:8080
```

Comment:
Missing new line, if you get the chance.

Reply:
Gah, saw this after merge. Will get it next time around.

Reply:
no worries